pyproject.toml

```diff
@@ -39,19 +40,33 @@ extra_standard_library = [
     "UserString",
 ]
 
+[tool.pyright]
+# Target oldest supported Python version
+pythonversion = "3.8"
+typeCheckingMode = "basic"
```

Comment:
Maybe we should set `typeCheckingMode` to `"standard"`?

Reply:
A lot of the additional errors are things disabled in typeshed as well. I've updated the config to run in "standard" mode and ignored new errors.